### PR TITLE
AMP-78 Add inline structural data to article

### PR DESCRIPTION
**Pull request by dosyara**
Added `itemscope` and `aditemProp` attributes to corresponding components of the article.
Tested via https://search.google.com/structured-data/testing-tool

### CODE REVIEW

File: src/app/components/article/article-meta-data.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    itemScope
+    itemType="https://schema.org/Organization"
+  >
+    <div itemProp="logo" itemScope itemType="https://schema.org/ImageObject">
+      <meta
+        itemProp="url"
+        content="https://www.economist.com/assets/the-economist-logo.png"
+      />
+    </div>
+    <meta itemProp="name" content="The Economis" />
```
Comment (yasgreen93) [inline src/app/components/article/article-meta-data.js]: Typo here.

File: src/app/components/article/article-publication-details.js [hunk 2]
```diff
@@ -102,8 +102,15 @@ const ArticlePublicationDetails = ({
...
     <StyledDateAuthor>
-      <StyledTime time={formatDate(new Date(datePublished))} />
-      {byline && ` | by ${byline}`}
+      <StyledTime itemProp="datePublished" content={getISODate(datePublished)}>
+        {formatDate(new Date(datePublished))}
+      </StyledTime>
+      {byline && (
+        <Fragment>
+          {' | by '}
+          <span itemProp="author">{byline}</span>
```
Comment (yasgreen93) [inline src/app/components/article/article-publication-details.js]: Can we extract this byline/title functionality into it's own little function or is that overkill if it's only being used here?

Comment (dosyara) [reply src/app/components/article/article-publication-details.js]: Agree, pushed a change

File: src/app/components/article/article-meta-data.js (hunk 1 above)
Comment (dosyara) [reply src/app/components/article/article-meta-data.js]: Thank you!

File: src/app/components/article/article-publication-details.js [hunk 3]
```diff
@@ -88,6 +88,17 @@ const StyledDateAuthor = styled('div')`
   letter-spacing: ${text.lineHeight.letterSpacing.sansOnStep['-3']};
 `;
 
+const ByLine = ({ byline }) => (
+  <Fragment>
+    {' | by '}
+    <span itemProp="author">{byline}</span>
+  </Fragment>
```
Comment (wajerrr) [inline src/app/components/article/article-publication-details.js]: When I was doing amp 74 (witch is pretty much the same exercise but in different format I have been tolad to hardcode author as 'The Economist' by @angelorepole )
so what i am doing now if there is no author coming from v1 I use economist
maybe worth checking ... https://github.com/EconomistDigitalSolutions/amp/compare/master...wajerrr:amp-28-with-helmet-parked?expand=1

**Pull request merged**